main.c

```diff
@@ -702,16 +719,17 @@ int main(void)
 	}
 
 	DBG_PRINTF_INIT();
-
-	uart_init(uart0, UART_BAUD);
-	gpio_set_function(UART_TX_PIN, GPIO_FUNC_UART);
-	gpio_set_function(UART_RX_PIN, GPIO_FUNC_UART);
-	uart_set_hw_flow(uart0, false, false);
+	
+	//uart_init(uart0, UART_BAUD);
+	//gpio_set_function(UART_TX_PIN, GPIO_FUNC_UART);
+	//gpio_set_function(UART_RX_PIN, GPIO_FUNC_UART);
+	//uart_set_hw_flow(uart0, false, false);
 
 	struct cmd_context ctx;
 	uint8_t uart_buf[(sizeof(uint32_t) * (1 + MAX_NARG)) + MAX_DATA_LEN];
 	ctx.uart_buf = uart_buf;
 	enum state state = STATE_WAIT_FOR_SYNC;
+	watchdog_reboot(0, 0, 4000); //Set WD to 4s
```

Comment:
4000 is a magic number, you should use 
`#define WD_TIMEOUT_MS (4000)`

Reply:
This watchdog reboot was a coding mistake and had it removed in the latest commit